internal/pkg/checks/check.go

```diff
@@ -104,19 +165,17 @@ func (c *Check) AddCheck(checks ...*Check) {
 	}
 }
 
-var Checks = &Check{
-	Name: "all",
-}
-
+// List this check and its child's description
 func (c Check) List() {
```

Comment:
struct functions should be consistent in usage of pointer and non-pointer forms

(use *Check for all)